src/cpp/dpm_sampler.cpp

```diff
@@ -526,7 +526,7 @@ void read_data(char* filename, std::ofstream& out_file)
     unsigned int seq = 0, i, j = 0, k = 0;
     out_file << "# reading data\n";
     if ((data = fopen(filename, "r")) == NULL) {
-        out_file << "Impossible to open file " << filename << "\n";
+        out_file << "Impossible to open file " << filename << std::endl;
```

Comment:
any reason you need `std::endl` here?

Reply:
I don't think so, and since I didn't change them all I can't argue about the style.

Reply:
Can you please change to `'\n'`?

Reply:
Done!